Makefile

```diff
@@ -101,3 +105,8 @@ set_env_class_prod:
 .PHONY: pipelines
 pipelines: ## Upload pipelines to Concourse
 	concourse/scripts/pipelines-bosh-cloudfoundry.sh
+
+.PHONY: env
+env: ## Display environment information
```

Comment:
Could you rename this to a verb like `showenv`?

`env` has quite a few meanings within our codebase and someone might expect `make dev env` to create them a development environment.